examples/platform_ios_headless.rs

```diff
@@ -0,0 +1,72 @@
+//! Defining an engine in Wasmer is one of the fundamental steps.
+//!
+//! This example builds on that of 'engine_headless.rs' but instead of
+//! serializing a module and then deserializing it again for your host machines target,
+//! We instead create an engine for our target architecture (In this case an ARM64 iOS device),
+//! serialize a simple module to a .dylib file that can be copied to an iOS project and
+//! deserialized/ran using the 'Headless C-API'.
+//!
+//! ```shell
+//! cargo run --example platform-headless-ios --release --features "cranelift"
+//! ```
+//!
+//! Ready?
+
+use std::fs::File;
+use std::path::Path;
+use std::str::FromStr;
+use tempfile::NamedTempFile;
+use wasmer::{imports, wat2wasm, Instance, Module, RuntimeError, Store, Value};
+use wasmer_compiler::{CpuFeature, Target, Triple};
+use wasmer_compiler_cranelift::Cranelift;
+use wasmer_engine_dylib::Dylib;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Let's declare the Wasm module with the text representation.
+    let wasm_bytes = wat2wasm(
+        r#"
+(module
+(type $sum_t (func (param i32 i32) (result i32)))
+(func $sum_f (type $sum_t) (param $x i32) (param $y i32) (result i32)
+local.get $x
+local.get $y
+i32.add)
+(export "sum" (func $sum_f)))
+"#
+        .as_bytes(),
+    )?;
+
+    // Create a compiler for iOS
+    let compiler_config = Cranelift::default();
+    let triple = Triple::from_str("x86_64-apple-ios")
```

Comment:
Shouldn't this be `aarch64-apple-ios` ? As far as I understand iOS runs only on the Apple silicon (ARM) ?

Reply:
Sorry yes. Accidentally changed when testing some things. `x86_64-apple-ios` is a thing though as it is the compile target for iOS simulator on Intel Macs.

Reply:
I've just fixed it